docs/next/components/mdx/MDXComponents.tsx

```diff
@@ -557,10 +630,10 @@ const TabGroup: React.FC<{children: any; persistentKey?: string}> = ({children,
           );
         })}
       </Tab.List>
-      <Tab.Panels>
+      <Tab.Panels ref={handleTabs}>
         {React.Children.map(children, (child, idx) => {
           return (
-            <Tab.Panel key={idx} className={classNames('p-3')}>
+            <Tab.Panel key={idx} className={classNames('p-3')} unmount={false}>
```

Comment:
Add a comment explaining the `unmount={false}` so no one accidentally removes it?